spyder/plugins/explorer/tests/test_explorer.py

```diff
@@ -46,5 +96,52 @@ def test_project_explorer(project_explorer):
     assert project_explorer
 
 
+@pytest.mark.parametrize('path_method', ['absolute', 'relative'])
+def test_copy_path(explorer_with_files, path_method):
+    """Test copy absolute and relative paths."""
+    project, _, file_paths, cb = explorer_with_files
```

Comment:
You should use ``__`` instead for a dummy variable, especially since we use ``_`` already for our gettext translation function.